src/options.h

```diff
@@ -2118,30 +2118,33 @@ nl_func_call_empty;
 extern Option<iarf_e>
 nl_func_call_start;
 
-// Whether to add a newline after '(' in a function call if '(' and ')' are in
-// different lines.
+// Whether to add a newline after '(' in a function call
+// if '(' and ')' are in different lines.
 extern Option<bool>
 nl_func_call_start_multi_line;
 
-// Whether to add a newline after '(' in a template.
+// Whether to add a newline after '<' of a template parameter list
+// if '<' and '>' are in different lines.
```

Comment:
NO, even in the same line, see the test

Reply:
Hmm, but if that's the case, shouldn't the option name just be `nl_template_start`?

Reply:
Yes